### PR TITLE
Feat/spaces

### PR DESCRIPTION
**Pull request by gregmartDOTin**

### CODE REVIEW

File: packages/spaces/src/lib/Spaces.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// Each exported component in the package should have its own stories file
+
+import type { Meta, StoryObj } from '@storybook/react';
+import { Spaces, SpacesProps } from './Spaces';
+
+const meta: Meta<typeof Spaces> = {
+  title: 'Components/Spaces/Spaces',
+  component: Spaces,
+  tags: ['autodocs'],
+};
```
Comment (LauRoxx) [inline packages/spaces/src/lib/Spaces.stories.tsx:10]: looks like you're missing the QueryClientProvider decorator

File: packages/spaces/src/lib/spaces-types.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+export interface SpacesReducerAction extends SpacesContextType {
+  type: 'SPACES' | 'ERROR' | 'LOADING';
+}
+
+export type ChildrenProps = {
+  spaces: NormalizedSpace[];
+};
+
+export type SpacesProps = {
+  query?: string;
```
Comment (LauRoxx) [inline packages/spaces/src/lib/spaces-types.tsx:75]: missing doc comments

File: packages/spaces/src/lib/Spaces.stories.tsx (hunk 1 above)
Comment (gregmartDOTin) [reply packages/spaces/src/lib/Spaces.stories.tsx:16]: Added a story.

File: packages/spaces/src/lib/spaces-types.tsx (hunk 2 above)
Comment (gregmartDOTin) [reply packages/spaces/src/lib/spaces-types.tsx:117]: Added comments.

**Review by jordan-a-young — APPROVED** [review]

**Pull request merged**